openfeature/openfeature_api.go

```diff
@@ -267,14 +415,13 @@ func initializer(provider FeatureProvider, evalCtx EvaluationContext) (Event, er
 	err := handler.Init(evalCtx)
 	if err != nil {
 		event.EventType = ProviderError
-		event.Message = fmt.Sprintf("Provider initialization error, %v", err)
+		event.Message = fmt.Sprintf("Provider initialization failed: %v", err)
```

Comment:
It was not a good practice of Golang error message, but it's probably better to keep it in case any automation or error checks exam the error message.